src/middlewared/middlewared/plugins/keychain.py

```diff
@@ -496,8 +513,13 @@ def generate_ssh_key_pair(self):
             "public_key": public_key,
         }
 
-    @api_method(KeychainCredentialRemoteSSHHostKeyScanArgs, KeychainCredentialRemoteSSHHostKeyScanResult,
-                roles=["KEYCHAIN_CREDENTIAL_WRITE"])
+    @api_method(
+        KeychainCredentialRemoteSSHHostKeyScanArgs,
+        KeychainCredentialRemoteSSHHostKeyScanResult,
+        roles=["KEYCHAIN_CREDENTIAL_WRITE"],
+        audit="Remote Host Key Scan:",
+        audit_extended=lambda data: data["host"]
```

Comment:
This also does not do any changes on the local system or expose any secrets, should it be audited?

Reply:
Probably not